flink-end-to-end-tests/flink-parent-child-classloading-test-program/src/main/java/org/apache/flink/streaming/tests/ClassLoaderTestProgram.java

```diff
@@ -52,48 +48,28 @@ public static void main(String[] args) throws Exception {
 
 				String gitCommitId;
 
-				try (InputStream propFile = ClassLoaderTestProgram.class.getClassLoader().getResourceAsStream(".version.properties")) {
+				try (InputStream propFile = ClassLoaderTestProgram.class.getClassLoader().getResourceAsStream("parent-child-test.properties")) {
```

Comment:
Should we also test the thread context classloader?

Reply:
We could do that, but we would basically test that the context classloader is being set correctly. I think we already cover that in unit tests.

Also, I want to keep this change to only fix what we currently have. 😅 

What do you think?

Reply:
No problem. I just wanted to mention that in case we want to test this as well.